.github/policies/issue.flag-for-triage.yml

```diff
@@ -15,5 +16,37 @@ configuration:
       then:
       - addLabel:
           label: 'Needs: Triage (Functions)'
+
+    - description: Add needs attention label to reopened issues
+      if:
+      - payloadType: Issues
+      - and:
+        - isAction:
+            action: Reopened
+        - not:
+            hasLabel: 'Needs: Attention :wave:'
+      then:
+      - addLabel:
+          label: 'Needs: Attention :wave:'
+
+    - description: Assign issues when marked for triage or when attention needed again
+      if:
+      - payloadType: Issues
+      - isAction:
+          action: Labeled
+      - not:
+          isAssignedToSomeone
+      - or:
+        - labelAdded:
+            label: 'Needs: Triage (Functions)'
+        - labelAdded:
+            label: 'Needs: Attention :wave:'
+      then:
+        - assignIcmUsers:
+            teamId: 114785
+            primary: true
+            secondary: false
```

Comment:
Casing changed to see if that can bring this one back (on top of the other changes noted in previous comment.